temporalio/workflow.py

```diff
@@ -940,7 +940,10 @@ def instance() -> Any:
 
 def in_workflow() -> bool:
     """Whether the code is currently running in a workflow."""
-    return _Runtime.maybe_current() is not None
+    try:
+        return _Runtime.maybe_current() is not None
+    except RuntimeError:
+        return False
```

Comment:
I think instead we should make `maybe_current` return `None` if there is no asyncio event loop (meaning catch the `RuntimeError` on the `get_running_loop` call there and return `None`). The only other place `maybe_current` is used is in the logger adapter, and we might as well not raise there either.